agent/agentcontainers/testdata/container_binds.json

```diff
@@ -0,0 +1,221 @@
+[
```

Comment:
Suggestion: I like grouping these tests under an additional subfolder containing the parent tests name, keeps the `testdata` folder a bit more maintainable.